Svg/Platform/Shared/WebRequestSvc.cs

```diff
@@ -23,5 +22,14 @@ public Stream GetResponse(Uri uri)
                 return await httpRequest.GetRequestStreamAsync();
             }).Result;
         }
+
+        private string EnsureFullPath(string path)
+        {
+            var fs = SvgEngine.Resolve<IFileSystem>();
+            var isRooted = fs.GetFullPath(path) == path;
```

Comment:
you call "fs.GetFullPath()" and pass in the _full URL_? (file://c/temp/myimage.png)?
Did you actually verify, that this works?

Can you please write some unit tests for this?
This should be trivial and _whenever we fix a bug_ we should ensure it cannot return - by writing a unit test.